README.md

```diff
@@ -23,6 +23,9 @@ If you're using this module, and need help, you can:
 If you are new to Seneca in general, please take a look at [senecajs.org][]. We have everything from
 tutorials to sample apps to help get you up and running quickly.
 
+### Seneca compatibility
+Supports Seneca versions from **0.6.5** up to **2.0.0**
```

Comment:
In test you used
SENECA_VER=@1.4.0
 SENECA_VER=@2.0.x
the readme is incorrect. It should say version 1.4.0 to 2.0.x